release/wait-for-image/wait-for-image.sh

```diff
@@ -21,16 +24,14 @@ check_not_empty \
 
 find_tag() {
     URL="https://quay.io/api/v1/repository/$1/tag?specificTag=$2"
-    {
-        if [ -z "$TOKEN" ]; then
-            gh_log notice "Connecting to Quay without token"
-            curl --silent --show-error --fail --location "$URL"
-        else
-            gh_log notice "Connecting to Quay with token"
-            curl --silent --show-error --fail --location "$URL" \
-                -H "Authorization: Bearer $TOKEN"
-        fi
-    } | jq -r ".tags[0].name"
+    CURL_PARAMS+=( "--silent" "--show-error" "--fail" "--location" "$URL" )
+    if [ -n "$QUAY_TOKEN" ]; then
+        >&2 gh_log notice "Connecting to Quay with token"
```

Comment:
Is the stderr redirect necessary now that you run `curl | jq` outside the block?

Reply:
Yes, as we later use the output of `find_tag`. This means that stdout must be clean

Reply:
Without `>&2`:

```
$ CI=false GITHUB_STEP_SUMMARY="a" ./release/wait-for-image/wait-for-image.sh stackrox-io/main:4.1.1                                                                   
Waiting 30 more seconds for stackrox-io/main:4.1.1...
```

With `>&2`:

```
$  CI=false GITHUB_STEP_SUMMARY="a" ./release/wait-for-image/wait-for-image.sh stackrox-io/main:4.1.1 
::notice::Connecting to Quay without token
::notice::Image 'stackrox-io/main:4.1.1' has been found on Quay.io.
```